### PR TITLE
Style migrated customer threads properly

### LINKED ISSUE #33430: BO > Customer threads - notifications lead to new page even though feature not enabled
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

> Posted by @Hlavtox 

### Point 1
`ps_faviconnotificationbo` has nothing to do with this, that's a different module.

### Point 2
This is NEW - `prestashop_8.1.1/admin-dev/index.php/sell/customer-service/customer-threads/10/view?_token=R1OEmlWsYSisbMDu2nh1loAsPcyjZq_IbX_IwDwUctI`
This is OLD - `prestashop_8.1.1/admin-dev/index.php?controller=AdminCustomerThreads&viewcustomer_thread=&id_customer_thread=10&token=669d85525063662641af830fa1d3c829`

The new page should be enabled only of the feature flag is enabled in New & Experimental features section. This works for backoffice, but not for notification center.

### Point 3
Display is broken on the new page, I will fix it.

➡️ I suggest separating the issues into two - 1) broken display, 2) notifications lead to new page even though feature not enabled.

---

When a customer send an Order's message, as Admin , I want to view the message by the module favicon-notifications, I will be redirect it to the page of **Customer service** > **Orders and messages timeline**, the timeline is badly displayed :x: 
While, if you access via BO>SELL>Cusomter Service>Customer Service, the timeline will be well displayed :heavy_check_mark: 

- It's caused by the old link provided by the module favicon-notifications whose redirect to : `prestashop_8.1.1/admin-dev/index.php/sell/customer-service/customer-threads/10/view?_token=R1OEmlWsYSisbMDu2nh1loAsPcyjZq_IbX_IwDwUctI`
- While, the new link provided by the button View in the page Customer Service is: `prestashop_8.1.1/admin-dev/index.php?controller=AdminCustomerThreads&viewcustomer_thread=&id_customer_thread=10&token=669d85525063662641af830fa1d3c829`

time-line-nn-okey

https://github.com/PrestaShop/PrestaShop/assets/92626242/8cb2e869-e0e5-44b1-934a-4e873cee9d8a

### Expected behavior

The Orders and messages timeline should be well displayed no matter the access was from the modal

image
Or from the BO Customer Service > Customer Service

### Steps to reproduce

1. FO > Click on customer account name > ORDER HISTORY AND DETAILS
2. Click on Details > Go to ADD A MESSAGE section > Send a message
3. Go to BO (Make sure you are not in Customer Service page)
4. Click on favicon notification > Messages
5. Right click on the message > open in a new tab
6. See error > Bad display :x:
time-line-nn-okey
7. Go to Customer Service > Customer Service > Click on View button in front of the message :heav
time-line-okey 

### PrestaShop version(s) where the bug happened

8.1.x & 8.1.0 & 8.1.1

### PHP version(s) where the bug happened

7.4

### If your bug is related to a module, specify its name and its version

ps_faviconnotificationbo

### Your company or customer's name goes here (if applicable).

_No response_

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Migrated customer page timeline properly styled and made responsive.
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Test this section now looks good even on mobile
| Fixed ticket?     | Fixes #33430 (only display issue)
| Related PRs       |
| Sponsor company   | 

### Desktop
desktop

### Mobile
mobile

### CODE REVIEW

File: admin-dev/themes/new-theme/scss/components/_timeline.scss [hunk 1]
```diff
@@ -1,123 +1,163 @@
...
+  }
+
+  .timeline-content {
+    padding: 1.25rem;
+    background-color: $white;
+    border: 1px solid #ddd;
+    @include border-radius(5px);
+  }
+
+  @media (max-width: 767px) {
```
Comment (kpodemski) [inline admin-dev/themes/new-theme/scss/components/_timeline.scss]: Please use helpers like `@include media-breakpoint-down(md) {`, etc.

Comment (Hlavtox) [reply admin-dev/themes/new-theme/scss/components/_timeline.scss]: @kpodemski Fixed, I don't know why I didn't do it the first time. It's even more precise.

**Review by FabienPapet — APPROVED** [review]

**Review by nicosomb — APPROVED** [review]

**Review by AureRita — APPROVED** [review]
Hi @Hlavtox 

Thank you for your PR, I tested it and it works, as we can see here :

[Untitled_ Aug 1, 2023 2_43 PM.webm](https://github.com/PrestaShop/PrestaShop/assets/100699445/32badde9-dc31-44da-a3a7-c6c95c0edb8c)

Automated link : 
https://github.com/AureRita/testing_pr/actions/runs/5726535286

The issue on it is legit so its QA :heavy_check_mark: 

Thank you

**Comment by nicosomb** [comment]
Thank you @Hlavtox !

**Pull request merged**